.github/PULL_REQUEST_TEMPLATE/back-end-pr-template-systems-intelligence.md

```diff
@@ -33,12 +33,13 @@
 <!--
     Migration Script: <link to migration script file>
     Description: <a description of the specific schema or data changes made by this migration script>
-
-    Keep this tag - the data enablement team will be notified via email of any changes: @nicheinc/data-enablement
```

Comment:
❓️ Wasn't the purpose of this tag to notify @nicheinc/data-enablement (which includes @ParryChen) that there's been a schema migration that might need to be reflected in Snowflake? I'm not necessarily opposed to submitting a Data Team Request in these cases, but it would be slightly more cumbersome for the PR author, vs. just linking the migration and uncommenting a tag.

Reply:
Hey so I think the structure of the data-enablement team has changed a lot, where I'm not part of the data enablement team anymore and these changes are now being shifted to the DEs(whom get all their work tracked through the data team request form). So I think these make sense!